server/src/main/java/org/opensearch/common/cache/service/NodeCacheStats.java

```diff
@@ -51,13 +52,15 @@ public void writeTo(StreamOutput out) throws IOException {
 
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        builder.startObject(NodesStatsRequest.Metric.CACHE_STATS.metricName());
```

Comment:
Yes, this was my mistake. In the response object, instead of having the stats in nodes.[node_id].caches.request_cache, it was at nodes.[node_id].request_cache. This change puts it in line with other nodes stats responses

Reply:
Assuming this new stats api is `experimental` and behind the feature flag ? Otherwise there will be bwc issues by changing the response format.

Reply:
Yes, it's both experimental and behind feature flag. If there were bwc issues, we would see them in the gradle check right? Or would we run into that later?